test/resources/tasks.test.ts

```diff
@@ -0,0 +1,65 @@
+import { Applicant, Task } from "onfido-node";
+
+import { exampleTask } from "../test-examples";
+import {
+  cleanUpApplicants,
+  cleanUpWebhooks,
+  createApplicant,
+  completeTask,
+  createWorkflowRun,
+  getExpectedObject,
+  onfido
+} from "../test-helpers";
+
+function getExpectedTask(exampleTask: Task, overrideProperties = {}) {
+  return getExpectedObject(exampleTask, {
+    task_def_id: expect.stringMatching(/^[0-9a-z-_]+$/),
```

Comment:
Is this a uuid4, correct?  I believe it should never contain a `_` 🤔

Reply:
No, it looks like this

```
{
        "created_at": "2024-03-19T09:59:51Z",
        "id": "upload_document_photo_024a2",
        "task_def_id": "upload_document_photo",
        "task_def_version": null,
```